### DISCUSSION TITLE
Install Kimai 2 for MetaFieldsBundle-1.15

### DISCUSSION THREAD

**Discussion by bleonaraya** [kimai/kimai#2251] [Questions & Answers]
Hello, good afternoon guys,

I am new to Kimai and development, currently I have a Hosting only with CPANEL access and not SSH. when I install in a root kimai-release-1.12 everything works OK, but I cannot use the component MetaFieldsBundle-1.15 (I JUST BUY THE COMPONENT OF 49 euros) because it is not compatible with the version.

given this download the kimai2-1.11 version but when uploading it by FTP the main page gives me a 500 SERVER error. that is, I can't run it directly as the kimai-release-1.12 version.

Please your help, I love this tool and I find it very useful. I just need it to be able to have the CUSTOM FIELDS plugin

**Comment by kevinpapst** [comment]
I have sent you an email. In short: plugins are not supported in FTP environments, because an installer has to be executed (which is only possible via SSH).

**Reply by bleonaraya** [reply]
Kevin Thank you very much for your answer and disposition,

Now it is more clear to me that the kimai-release-1.12 version is fine but for the add-ons, it is NOT compatible with FTP.

I will see the method that my Hosting provider can grant me access to install by SSH.

have a nice evening.
